lib/pavilion/resolver/resolver.py

```diff
@@ -636,7 +735,9 @@ def _load_raw_configs(self, request: TestRequest, modes: List[str],
 
             # Apply modes.
             try:
-                test_cfg = self.apply_modes(test_cfg, modes)
+                test_cfg = self.apply_modes(test_cfg, modes, request.suite)
+                test_cfg = self.apply_host(test_cfg, self._host, request.suite)
+                test_cfg = self.apply_os(test_cfg, self._os, request.suite)
```

Comment:
The stack should be:
 - modes
 - test_layers
 - host
 - os
 - <empty config>

What you have is very backwards.

Reply:
IE settings in `host` should override `os`, and so on.